pluma/stream/zeromq.py

```diff
@@ -206,4 +226,30 @@ def __init__(self, eventcode: int, **kw):
                  ('Subject.Position.Y', np.single),
                  ('Point.Position.X', np.single),
                  ('Point.Position.Y', np.single)]],
+            **kw)
+class ProtocolNewSceneStream(ZmqStream):
+    def __init__(self, eventcode: int, **kw):
+        super(UnityNewSceneStream, self).__init__(
+            eventcode,
+            streamtype=StreamType.UNITY,
+            filenames=[
+                'Protocol/NewScene_Frame1.bin',
+                'Protocol/NewScene_Frame2.bin'],
+            dtypes=[
+                [('Timestamp', np.ulonglong)],
+                [('SceneType', np.intc),
+                 ('SpawnID', np.intc),
+                 ('SceneDuration', np.intc)]],
+            **kw)
+class ProtocolITIStream(ZmqStream):
+    def __init__(self, eventcode: int, **kw):
+        super(UnityITIStream, self).__init__(
```

Comment:
```suggestion
class ProtocolItiStream(ZmqStream):
    def __init__(self, eventcode: int, **kw):
        super(ProtocolItiStream, self).__init__(
```

Spacing to separate class definitions and rename class to follow [capitalization conventions](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers).

Reply:
solved in 2eae7bf